packages/components/tabs/src/tab-nav.tsx

```diff
@@ -308,7 +309,9 @@ const TabNav = defineComponent({
         : null
 
       const tabs = props.panes.map((pane, index) => {
-        const tabName = pane.props.name || pane.index || `${index}`
+        const tabName = isNumber(pane.props.name)
+          ? pane.props.name
+          : pane.props.name || pane.index || `${index}`
```

Comment:
```suggestion
       const tabName = pane.props.name ?? pane.index ?? `${index}`
```

Is it more convenient to use `??`, at the same time the `pane.props.name` should not have a default value `''` (the default value should be `index` of TabPane).

Reply:
Yes, I also used it at the beginning `? ?` , the review coach at the time let me use the `function`, and I will adjust it.
Thanks for your review.